packages/admin-api-client/package.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "@shopify/admin-api-client",
+  "version": "0.0.1",
+  "description": "Shopify Admin API Client - A lightweight JS client to interact with Shopify's Admin API",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/shopify-api-js.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "main": "./dist/umd/admin-api-client.min.js",
```

Comment:
I wonder if we should point `main` to the `cjs` index file, since we're not building UMD any more, right?